src/main/java/Frenchie.java

```diff
@@ -79,6 +81,16 @@ public static void main(String[] args) {
                                 target_task.toString() + "\n" +
                                 "____________________________________________________________");
                     }
+                } else if (input.contains("delete")){
+                    String[] parts = input.split(" ");
+                    int index = Integer.parseInt(parts[1]) - 1;
+                    Task target_task = frenchie.tasks.get(index);
+                    frenchie.deleteTask(index);
+                    System.out.println("____________________________________________________________\n" +
+                            "Noted. I've removed this task: \n" +
+                            target_task.toString()   + "\n" +
+                            "Now you have " + frenchie.getNumOfTasks() + " tasks in the list.\n" +
+                            "____________________________________________________________");
```

Comment:
As the number of keywords increase, perhaps you might want to start looking at enums to have a collection of your keywords!